roles/bwc/templates/rbac_assignments/assignments.yml.j2

```diff
@@ -0,0 +1,5 @@
+---
+
+  username: {{ item.name }}
+  roles:
+    {{ item.roles | to_nice_yaml }}
```

Comment:
Same as noted in https://github.com/StackStorm/ansible-st2/pull/126/files#r106033276 this should be more correct way:
`{{ item.roles | to_nice_yaml(2) | indent(2) }}`

in case when `item.roles` contains more complex data.